core/src/main/scala/io/finch/Finch.scala

```diff
@@ -0,0 +1,16 @@
+package io.finch
+
+import cats.effect.Effect
+import io.finch.endpoint.effect.EffectEndpoints
+import io.finch.syntax.effect.EffectEndpointMappers
+
+abstract class Finch[F[_]](implicit val E: Effect[F])extends EffectEndpoints[F]
+  with Outputs
+  with ValidationRules
+  with EffectInstances[F] {
+
+  type Endpoint[A] = io.finch.Endpoint[F, A]
```

Comment:
Do we need this?

Reply:
Again, it's just to make life easier for customers. In some cases, I believe it wouldn't even require to change anything besides the `import` line:
```scala
val helloWorld: Endpoint[String] = get("/hello") { Ok("hello world") }
//instead of
val helloWorld: Endpoint[MyFancyEffect, String] = get("/hello") { Ok("hello world") }
```